sw/device/sca/sha3_serial.c

```diff
@@ -585,10 +553,11 @@ bool test_main(void) {
   simple_serial_register_handler('k', sha3_serial_set_key);
   simple_serial_register_handler('p', sha3_serial_single_absorb);
   simple_serial_register_handler('b', sha3_serial_batch);
-  simple_serial_register_handler('t', sha3_serial_fixed_key_set);
+  simple_serial_register_handler('t', sha3_serial_fixed_message_set);
   simple_serial_register_handler('l', sha3_serial_seed_lfsr);
+  simple_serial_register_handler('m', sha3_set_masks);
```

Comment:
We should also add this function to the comment at line 22 of the file "Set mode/mask_off ('m')"